app/views/user_mailer/shared/_non_discrimination_taglines.html.erb

```diff
@@ -0,0 +1,24 @@
+<html>
+<body>
+<div>
+  <h2><b> Discrimination is against the law </b></h2><br>
+  The <%= site_short_name %> complies with applicable federal civil rights laws and does not discriminate on the basis of race, color, national origin, age, disability, or sex. The <%= site_short_name %> does not exclude people or treat them differently because of race, color, national origin, age, disability, or sex.<br>
+  <h3><b>Services available through the Health Connector</b></h3>
+  The <%= site_short_name %> provides free aids and services to people with disabilities to communicate effectively with us. These aids and services include:<br><br>
+  <ul style="list-style-type:circle">
+    <li>Qualified sign language interpreters</li>
+    <li>Written information in other formats like large print, audio, accessible electronic formats, and other formats.</li>
+  </ul><br><br>
+  The <%= site_short_name %> provides free language services to people whose primary language is not English. These services include:<br>
+  <ul style="list-style-type:circle">
+    <li>Qualified interpreters</li>
+    <li>Information written in other languages</li><br><br>
+  If you need these services, contact the <%= site_long_name %> at 888-813-9220  TTY: 711 for people who are deaf, hard of hearing, or speech disabled.<br><br>
+  <h2><b>You can file a grievance if you think you have been discriminated against</b></h2><br><br>
+  If you believe that the Health Connector or another party operating on the exchange has failed to provide these services or discriminated in another way on the basis of race, color, national origin, age, disability, or sex, you can file a grievance with the <%= site_short_name %> or the U.S. Department of Health and Human Service Office for Civil Rights.<br><br>
+  If you want to file a grievance with the <%= site_short_name %>, please visit our website to download our policy and complaint form. You can find these documents online at: www.MAhealthconnector.org/site-policies/nondiscrimination-notice<br><br>
+  If you need help filing a grievance, please email the <%= site_short_name %>’s Compliance Manager at Nondiscrimination@state.ma.us or call 617-936-1037.<br><br>
```

Comment:
nondiscrimination@state.ma.us and the phone # should be settings and methods as well